flake.nix

```diff
@@ -110,6 +110,15 @@
           type = "app";
           program = "${self.packages.${system}.default}/bin/llama-server";
         };
+        apps.llama-server-openai-proxy = {
+          type = "app";
+          program = "${
+            (let pythonWithPkgs = pkgs.python3.withPackages (ps: with ps; [ flask requests ]);
+             in pkgs.writeShellScriptBin "run-openai-proxy" ''
+                  ${pythonWithPkgs}/bin/python3 ${self}/examples/server/api_like_OAI.py
+             '')
+          }/bin/run-openai-proxy";
```

Comment:
Can't suggest anything concrete right now, but it might be a good thing to do some form of `buildPythonApplication` because its probably less affected by PYTHONPATH contamination.